presto-elasticsearch/src/main/java/io/prestosql/elasticsearch/client/ElasticsearchClient.java

```diff
@@ -449,15 +450,17 @@ public IndexMetadata getIndexMetadata(String index)
                     mappings = mappings.elements().next();
                 }
 
-                return new IndexMetadata(parseType(mappings.get("properties")));
+                JsonNode metaNode = nullSafeNode(mappings, "_meta");
+
+                return new IndexMetadata(parseType(mappings.get("properties"), nullSafeNode(metaNode, "presto_meta")));
```

Comment:
Let's call this just `presto`. It's understood it's metadata, as it appears in the `_meta` section.